software-table.tex

```diff
@@ -45,6 +45,8 @@
 \stdpopsim &
     A community-maintained collection of empirical data for species
     and demographic models specified in Demes
-    format~\citep{adrion2020community}.\\
+    format~\citep{adrion2020community}.
+    \ggcomment{stdpopsim doesn't yet have Demes support}
```

Comment:
This has been on the to-do list at stdpopsim for a while. Any case we can at least make some progress on that in stdpopsim before submitting so we can reasonably include stdpopsim in the table here?

Reply:
That does seem like a good idea.